Doc/library/socket.rst

```diff
@@ -827,8 +827,8 @@ The following functions all create :ref:`socket objects <socket-objects>`.
    ``(host, port)``) and returns the socket object.
 
    *family* should be either :data:`AF_INET` or :data:`AF_INET6`.
-   *backlog* is the queue size passed to :meth:`socket.listen`; if not specified
-   , a default reasonable value is chosen.
+   *backlog* is the queue size passed to :meth:`socket.listen`; if not
+   specified, a default reasonable value is chosen.
```

Comment:
Small formatting fix for [this](https://docs.python.org/dev/library/socket.html#socket.create_server):

![image](https://github.com/python/cpython/assets/46876382/5091d568-6ac1-47a5-ac5b-7c8befaeb67b)